compiler_opt/rl/constant_value_network.py

```diff
@@ -47,6 +47,6 @@ def __init__(self, input_tensor_spec, constant_output_val=0, name=None):
 
   def call(self, inputs, step_type=None, network_state=(), training=False):
     _ = (step_type, training)
-    shape = nest_utils.get_outer_array_shape(inputs, self._input_tensor_spec)
-    return tf.constant(
-        self._constant_output_val, tf.float32, shape=shape), network_state
+    shape = nest_utils.get_outer_shape(inputs, self._input_tensor_spec)
```

Comment:
does it change the code behavior?

Reply:
the output of the network is the same as before (same shape + constant value), however this implementation is potentially slightly slower because it checks the dynamic shape of the input rather than the static shape. I haven't noticed this being an issue though - the policy network is still significantly slower in comparison.